frontend/tests/fixtures.ts

```diff
@@ -112,6 +114,29 @@
     await deleteUser(context.request, tempUser.id);
     await context.close();
   },
+  tempUserInTestOrg: async ({browser, page, mailboxFactory}, use, testInfo) => {
+    const mailbox = await mailboxFactory();
+    const email = mailbox.email;
+    const name = `Test: ${testInfo.title} - ${email.replaceAll('@', "(at)")}`;
+    console.log(name);
+    const password = email;
+    const tempUserId = await registerUser(page, name, email, password);
+    await loginAs(page.request, 'admin');
+    await addUserToOrg(page.request, tempUserId, testEnv.testOrgId, OrgRole.User);
+    await loginAs(page.request, email, password);
```

Comment:
why do we login as the user being created? I'd rather keep this the same as tempUser where you're not logged in as them.

Reply:
Because the corresponding `tempUser` fixture ends up with a login cookie for the just-created user (thanks to the `registerUser` call just before `await use(tempUser)`), and I wanted this one to behave the same way. But since this fixture has `loginAs('admin')` in between, it would result in *different* test behavior than `tempUser`, and I wanted to keep the same ends-up-logged-in behavior that `tempUser` has.

But if we don't want to document that `tempUser` ends up logged in as that user, then sure, we can get rid of this login as well. Let me know what you think.

Reply:
no that sounds good, I didn't realize that the normal temp user worked that way, but that makes sense now